libdevcore/Common.h

```diff
@@ -294,6 +294,12 @@ enum class WithExisting: int
 	Kill
 };
 
+/// Get the current time in seconds since the epoch in UTC
+inline time_t getNowUTC()
```

Comment:
I'd rather leave it returning `time_t `as this is what is already returned by the time() function. Why move the implementation? It's quite a small function so I would suppose it's a prime candidate for inlining.